package.json

```diff
@@ -21,9 +24,13 @@
   },
   "homepage": "https://github.com/gavinaiken/eslint-plugin-chai-expect-keywords",
   "bugs": "https://github.com/gavinaiken/eslint-plugin-chai-expect-keywords/issues",
+  "engines": {
+    "node": ">=6.0.0"
```

Comment:
I'm using 10 or 12 in all my projects now so would be fine for me to go even higher, although probably keeping the support as broad as possible would benefit the general community. Probably bumping this to 8 would be a good place to land, although 6 is fine with me too.

Reply:
Ok, bumping to 8 is nice for the features that could be used. Could do a little ES6 refactoring (some would work with just 6, but 8 is better of course).

Reply:
I've merged this one now, could bump up further and use ES6 features later

Reply:
Or if you don't mind, if I could bump to 8.3, as that started support for object rest properties which can be very handy in such refactoring.